.docs/test_plan.md

```diff
@@ -70,10 +70,10 @@ These tests are dependent of the configuration of the underlying infrastructure
 |        |       |
 | ------ | ----- |
 | **Date** | 03/29/2023 |
-| **Infra version** |  2.12.0  |
-| **Core version** | 0.11.1 |
```

Comment:
Was there any value for the previous versions ?
It would have been great to be able to make a comparison across time.

Reply:
i'll do a document for each version tested. I propose to create a folder in main with all the documents.

Reply:
@ngruelaneo Indeed. The file test_plan is the plan no current version and excution value should inside. It's the plan file. 
@aneojgurhem Good Idea the diff time could should be another file where we could see the delta between the current version and previous release tagged in armonik-version.txt What do you think ?
@YKharouni in the test_plan you can add a link to this new diff time file